go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -645,7 +645,7 @@ func (be *BuiltinBackupEngine) backupFiles(
 			name := fmt.Sprintf("%v", i)
 			err := be.backupFile(ctxCancel, params, bh, fe, name)
 			if err != nil {
```

Comment:
We could make the relationship even more obvious this way:
```suggestion
			if err := be.backupFile(ctxCancel, params, bh, fe, name); err != nil {
```